orchestration/connectionmanager/st2.py

```diff
@@ -61,6 +61,17 @@ def list_actions(self, pack_name=''):
                     action_dict[elem['ref']] = elem
         return action_dict
 
+    # Function to get the action
+    # Caller can store the result in DB
+    # @Input: PackName, ActionId
+    # @output: the ActionDetails
+    def get_action(self, id, pack_name=''):
+        auth_token = self.authenticate()
+        headers = {'X-Auth-Token': auth_token}
+        url = Orchconstants().get_st2_action_list_url(self.server) + '/' + id
+        response = requests.get(url, headers=headers, verify=False)
+        return response.json()
```

Comment:
Please return response.status_code, response.json()
In the caller just check if return code is != Apiconstants.HTTP_OK then return the error in response with the proper HTTP error code we got

Reply:
Done